### ISSUE TITLE
Re-balance moonstone armor

### ISSUE THREAD

**Issue by ChloeDawn** [MysticMods/Druidcraft#99]
**Describe your suggestion:**
The moonstone armor is currently better than netherite armor, which seems a little over-powered

**Describe alternatives you've considered:**
Either a direct re-balance, or a config to allow pack makers to determine the armor values

**Comment by dmsw0303** [comment]
yep,  this is ultimately why I ended up just having to remove this from my pack
completely destroys most the game progression for no reason

**Comment by ChloeDawn** [comment]
You could disable the recipes with CraftTweaker instead of removing the entire mod, that's what I did. ^^

**Comment by noobanidus** [comment]
Or better yet, if you don't want to install CraftTweaker, you can always use a Vanilla data pack to disable the recipe.

**Comment by wildcardgamez** [comment]
IMO, it works fine, but maybe it should have a change? Maybe its protection could depend on the phase of the moon, and it could be better than netherite only on full or gibbous moons.
